cmd/controller-manager/controller-manager.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	bucketcontroller "github.com/kubernetes-sigs/container-object-storage-interface-api/controller"
+	"github.com/kubernetes-sigs/container-object-storage-interface-controller/pkg/bucketrequest"
+
+	"github.com/golang/glog"
+)
+
+var ctx context.Context
```

Comment:
No need for this global I believe, see below.

Reply:
Fixed.